test/inheritAttributes.test.js

```diff
@@ -2065,3 +2066,175 @@ QUnit.test('Test to check use of either Segment Template or Segment List when bo
   assert.equal(actual.length, 1);
   assert.deepEqual(actual, expected);
 });
+
+QUnit.test('keySystem info for representation - lowercase UUIDs', function(assert) {
```

Comment:
thanks for writing tests!

Reply:
No problem. I was wondering how thorough I should have been (all available keySystems, empty ContentProtection xml segment etc,,,)

Reply:
I think this is good enough.